dnsapi/dns_maradns.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env sh
+
+#Usage: dns_maradns_add _acme-challenge.www.domain.com "token"
+dns_maradns_add() {
+  fulldomain="$1"
+  txtvalue="$2"
+
+  MARA_ZONE_FILE="${MARA_ZONE_FILE:-$(_readaccountconf_mutable MARA_ZONE_FILE)}"
+  MARA_DUENDE_PID_PATH="${MARA_DUENDE_PID_PATH:-$(_readaccountconf_mutable MARA_DUENDE_PID_PATH)}"
+
```

Comment:
why not check if `MARA_DUENDE_PID_PATH` is empty ?
And for MARA_DUENDE_PID_PATH ?

Like this   https://github.com/Neilpang/acme.sh/blob/master/dnsapi/dns_cf.sh#L19

Reply:
`MARA_ZONE_FILE` is checked by `_check_zone_file`. I've elaborated that check and added a similar check for `MARA_DUENDE_PID_PATH` as well, thanks for pointing out!